app.js

```diff
@@ -26,4 +26,235 @@ about.forEach((n) => {
+  const pDescription = document.createElement('p');
+  const languageUl = document.createElement('ul');
+  const projectBtn = document.createElement('div');
+
+  // initialise elements
+  // card image
+  projectImageWeb.src = element.featuredImg;
+  projectImageWeb.classList.add('project-image-web');
+  projectImageWeb.alt = 'project-image';
+  projectImageMobile.src = element.featuredImgMobile;
+  projectImageMobile.classList.add('first-project-image');
+  projectImageMobile.alt = 'First-project';
+  // cardheader
+  projectHeader.classList.add('project-heading');
+  projectHeader.innerText = element.name;
+  // stact text
+  canopyText.innerText = 'canopy';
+  canopyText.classList.add('content-text', 'canopy');
+  countImage.src = './images/Counter.svg';
+  countImage.classList.add('count-vector');
+  backText.innerText = 'Back end dev';
+  backText.classList.add('content-text');
+  countImage2.src = './images/Counter.svg';
+  countImage2.classList.add('count-vector');
+  yearText.innerText = '2015';
+  yearText.classList.add('content-text');
+  pDescription.innerText = 'A daily selection of privately personalized reads; no accounts or sign-ups required';
+  languageUl.classList.add('languages');
+  // place elements in document
+  projectDiv.classList.add('project');
+  projectDiv.append(projectImageDiv, projectContentDiv);
+  projectImageDiv.classList.add('project-image');
+  projectImageDiv.append(projectImageMobile, projectImageWeb);
+  projectContentDiv.classList.add('project-content');
+  projectContentDiv.append(projectHeader, projectBody);
+  projectBody.classList.add('project-body');
+  projectBody.append(projectDescription, pDescription, languageUl, projectBtn);
+  projectDescription.classList.add('description');
+  projectDescription.append(
+    canopyText,
+    countImage,
+    backText,
+    countImage2,
+    yearText,
+  );
+
+  projects[i].technologies.forEach((tech, n) => {
+    const languageLi = document.createElement('li');
+    languageLi.classList.add('language-used');
+    languageUl.append(languageLi);
+    languageUl.classList.add('languages-ul');
+    languageLi.innerText = projects[i].technologies[n];
+  });
+
+  projectContainer.appendChild(projectDiv);
+
+  projectBtn.innerHTML = `<button id="${i}" data-modal-target="#modal" class="btn project-link" >See project</button>`;
+  projectDiv.classList.add('card-reverse');
+
+  // reverse  card
+  if (i % 2 !== 0) {
+    projectDiv.style.flexDirection = 'row-reverse';
+  } else {
+    projectDiv.style.flexDirection = 'row';
+  }
+});
+
+// Menu PopUP
+const modalContainer = document.querySelector('.popup');
+const openModalButton = document.querySelectorAll('[data-modal-target]');
+const overlay = document.getElementById('overlay');
+
+function creatPopup(imageSource) {
+  const modal = document.createElement('div');
+  modal.innerHTML = `<div class="modal active" id="modal">
+  <div class="modal-header">
+    <h4 class="modal-heading">Tonic</h4>
```

Comment:
Good job working on the details popup window 😃, as we can see, your images are generating dynamically but you are not generating all contents dynamically like `name`, `description`, `technologies`, `liveVersion`, and `sourceCode` as it is [required](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/8_popup.md#:~:text=You%20need%20to%20update%20the%20main%20page%20so%20that%20the%20projects%20section%20is%20created%20dynamically%20using%20the%20information%20stored%20in%20that%20JavaScript%20object.%20Remember%2C%20all%20of%20the%20HTML%20in%20that%20section%20is%20created%20when%20the%20page%20loads.) 😄. 

- [ ] Kindly make sure the information above is generated dynamically because all projects can't have the same data